src/_P099_XPT2046Touch.ino

```diff
@@ -512,14 +512,16 @@ boolean Plugin_099(uint8_t function, struct EventStruct *event, String& string)
 
                 if (!bitRead(P099_CONFIG_FLAGS, P099_FLAGS_SEND_Z) && validDeviceIndex(DeviceIndex)) { // Do NOT send a Z event for each
                                                                                                        // touch?
-                  Device[DeviceIndex].VType      = Sensor_VType::SENSOR_TYPE_DUAL;
-                  Device[DeviceIndex].ValueCount = 2;
+                  // FIXME TD-er: Should not change anything in the Device vector.

```

Comment:
I'm pretty sure that option to disable the Z value can be removed, so this can be fixed. I'll have a look at that code.

Reply:
Yeah, was not sure how to name that function.
I was tempted a bit to call it something like "I_know_this_should_not_be_changed_but_I_am_stubborn()`
But in the end I named it a bit more 'professional looking' :)